apps/prairielearn/src/server.js

```diff
@@ -1534,6 +1534,32 @@ export async function initExpress() {
     '/pl/course_instance/:course_instance_id(\\d+)/assessments',
     (await import('./pages/studentAssessments/studentAssessments.js')).default,
   );
+  // Client files for assessments
```

Comment:
@mylesw27 for posterity, can you share the rationale for re-ordering these routes? If you can phrase it as a comment to leave in the source code "These routes must come before XYZ because...", even better!

Reply:
Yes, we needed to reorder these routes to avoid hitting the middleware in` .../assessment/:assessment_id` that would redirect to an assessment instance if one existed. I have added a comment about this in [a9f667c](https://github.com/PrairieLearn/PrairieLearn/pull/10889/commits/a9f667c61e9bff457a4a1f87f8713b743c7c1414).

Reply:
Perfect, thank you!